Functions/Tfun.py

```diff
@@ -36,8 +36,13 @@ def t_fun(environment, node):
     parameter = node["elements"]["e1"]
     parameter_type = node["elements"]["e2"]
 
+    if parameter_type != TYPE.BOOL and parameter_type != TYPE.INT:
```

Comment:
o que tá acontecendo aqui

Reply:
Se não for um tipo já final, tem que inferir tipo dele